tools/cbmc/proofs/ARP/ARP_FreeRTOS_OutputARPRequest/OutputARPRequest_harness.c

```diff
@@ -0,0 +1,27 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_ARP.h"
+
+
+ARPPacket_t arp_packet;
+NetworkBufferDescriptor_t pxNetworkBuffer;
```

Comment:
If `NetworkBufferDescriptor_t` is not a pointer type, please call this `xNetworkBuffer`